### DISCUSSION TITLE
Getting a Credly badge and winning a prize

### DISCUSSION THREAD

**Discussion by zawhtut** [microsoft/hack-together#55] [General]
> There is a list of [📃 Top Microsoft Graph Scenarios](https://github.com/microsoft/hack-together/blob/main/top-scenarios.md) available! You may build one of these scenarios to win a digital badge, or get inspired and build your own project ideas!

According to that statement, do I have to use the proposed scenario to win the prizes?

Sincerely,
Zaw

**Comment by waldekmastykarz** [comment]
You don't have to use a proposed scenario. We're included them to spark inspiration, but you're free to build a hack around any other use cases you'd like. All projects submitted by March 15, qualify for a digital badge as long as they're built using .NET and use the Microsoft Graph .NET SDK.
